Frontend/src/components/rankings/DailyRanking.vue

```diff
@@ -9,17 +9,17 @@
               v-for="(flight, index) in flights.slice(0, maxRows)"
               v-bind:item="flight"
               v-bind:index="index"
-              v-bind:key="flight._id"
-              @click="routeToFlight(flight.flightId)"
-              @mouseover="updateHighlightedFlight(flight.flightId)"
+              v-bind:key="flight.id"
+              @click="routeToFlight(flight.id)"
+              @mouseover="updateHighlightedFlight(flight.id)"
               @mouseleave="updateHighlightedFlight(null)"
             >
               <td scope="row">{{ index + 1 }}</td>
-              <td>{{ flight.pilot }}</td>
-              <td>{{ flight.takeoff }}</td>
-              <td>{{ flight.distance }} km</td>
-              <td>{{ flight.taskType }}</td>
-              <td>{{ flight.points }} P</td>
+              <td>{{ flight.User.name }}</td>
+              <td>{{ flight.takeoff.name }}</td>
```

Comment:
* Den Zusatze "Start" bzw. "Startplatz" sollten wir entfernen
* Die Zelle mit User.name und Takeoff.name sollten Links auf die entsprechenden Seiten sein

Reply:
Genau, das müsste dann schon von der API so kommen. Ob das Start oder Landeplatz ist könnte ja eine weiter Zeile in der DB sein. z.B. Detail
Ein klick auf eine Zeile führt zum Flug. Keine clickzones für Takeoff, Name etc

Reply:
Habe von Sascha auch nur die Startplätze importiert.
Die Landeplätze werden ja eh über die Google Maps API bezogen.
Streiche ich dann raus.

Reply:
Perfekt!